doc/stdenv/stdenv.chapter.md

```diff
@@ -837,6 +837,10 @@ The distribution phase is intended to produce a source distribution of the packa
 
 #### Variables controlling the distribution phase {#variables-controlling-the-distribution-phase}
 
+##### `doDist` {#var-stdenv-doDist}
+
+Set to true to enable the distribution phase.
```

Comment:
```suggestion
If set, the distribution phase is executed.
```

Because it follows the "convention" of the variable `dontCopyDist` defined below.